zerver/lib/user_groups.py

```diff
@@ -237,8 +237,15 @@ def create_system_user_groups_for_realm(realm: Realm) -> Dict[int, UserGroup]:
         realm=realm,
         is_system_group=True,
     )
+    nobody_system_group = UserGroup(
+        name=UserGroup.NOBODY_GROUP_NAME,
+        description="Nobody in this organization",
```

Comment:
I think "Nobody" would be a clearer label for this, since it's not like there's other users outside the organization who will have access.

Reply:
Do we want to update the name or description here?

Reply:
The name `@role:nobody` is fine I think; just the description.

Reply:
Updated the description.